core/src/com/unciv/ui/worldscreen/mainmenu/OptionsPopup.kt

```diff
@@ -294,27 +292,39 @@ class OptionsPopup(val previousScreen: BaseScreen) : Popup(previousScreen) {
                 }
                 if (modLinks.isNotOK()) noProblem = false
                 lines += FormattedLine()
+                if (noProblem) lines += FormattedLine("No problems found.".tr())
+
+                postCrashHandlingRunnable {
+                    // When the options popup is already closed before this postRunnable is run,
+                    // Don't add the labels, as otherwise the game will crash
+                    if (stage == null) return@postCrashHandlingRunnable
+                    // Don't just render text, since that will make all the conditionals in the mod replacement messages move to the end, which makes it unreadable
+                    // Don't use .toLabel() either, since that activates translations as well, which is what we're trying to avoid,
+                    // Instead, some manual work needs to be put in.
+
+                    val expanderTab = ExpanderTab(mod.name, startsOutOpened = false){
+                        it.defaults().align(Align.left)
+                        for (line in lines) {
+                            val label = if (line.starred) Label(line.text + "\n", BaseScreen.skin)
+                                .apply { setFontScale(22 / Fonts.ORIGINAL_FONT_SIZE) }
+                            else Label(line.text + "\n", BaseScreen.skin)
+                                .apply { if (line.color != "") color = Color.valueOf(line.color) }
+                            label.wrap = true
+                            it.add(label).width(stage.width / 2).row()
+                        }
+                    }
+
+                    val loadingLabel = modCheckResultTable.children.last()
+                    modCheckResultTable.removeActor(loadingLabel)
+                    modCheckResultTable.add(expanderTab).row()
+                    modCheckResultTable.add(loadingLabel).row()
+                    modCheckCheckBox!!.enable()
```

Comment:
Then one last question: should this enabling of the mod check checkbox be done for each mod, or only once at the end? Seeing as its only disabled once outside the entirety of this thread I'd have assumed the latter

Reply:
Oh yes that's a mistake :)